### PR TITLE
Add kaamiki setup and update README.md

### PR DESCRIPTION
**Pull request by stardust-xs**
_Oct 11, 2020 - v0.0.1_

**Added:**
- Author name in license block in **README.md**.
- Necessary files for setting up (installing) @kaamiki on other machines.

### CODE REVIEW

**Comment by stardust-xs** [comment]
Please review [this](https://github.com/kaamiki/kaamiki/pull/2/commits/c14a71651c43a632799ac659af5a2c3258a01018) commit @PranaliRPatil , @kwattorama and @srushti-maladkar.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        "License :: OSI Approved :: Apache Software License",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3 :: Only",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Topic :: Home Automation",
+        "Topic:: Security",
+        "Topic:: Security:: Cryptography",
```
Comment (srushti-maladkar) [inline setup.py]: Spaces mismatch
Expected "Topic :: Security" to maintain uniform format

**Review by srushti-maladkar — CHANGES_REQUESTED** [review]
Comments to be resolved before merge

**Review by PranaliRPatil — APPROVED** [review]
Looks fine.

File: setup.py (hunk 1 above)
Comment (stardust-xs) [reply setup.py]: Fixed [here](https://github.com/kaamiki/kaamiki/commit/0ca87ca7a5df97ed3f246591da8c2ae0d63a2afe), please check.

**Review by srushti-maladkar — APPROVED** [review]
Approved

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+As community members, contributors and maintainers of this project, we are thankful and commit to respect all people who participate in the community through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+
+## Our standards
+
+- We welcome and encourage participation by everyone and are committed to being in a community where everyone feels good about joining.
+- We sacredly believe in the concept of **[Vasudhaiv Kutumbkam](https://en.wikipedia.org/wiki/Vasudhaiva_Kutumbakam)** (The World is One Family) in letter and spirit. Although we may not be able to satisfy everyone, we will always work to treat everyone well.
+- We don't encourage support and suggestions on demand, we expect everyone to be proactive. We believe in proactive help or contribution to a problem. While doing so, please be careful not to be too patronizing or disrespectful. When someone turns to you for help, always be open, listen and try to put yourself in the position of your counterpart. If you don't have time, let the person know or maybe refer to a team member who is just as familiar with the topic.
```
Comment (kwattorama) [inline CODE_OF_CONDUCT.md]: this point will relate more in the expected behavior section

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+- The Code of Conduct applies in particular to team interaction, including all events hosted/conducted by Kaamiki, chat and video conferences, social media communication, pull request feedback, participation in open source projects, communication with our team at conferences or events where we represent Kaamiki, whether offline or online.
+
+## Conflict Resolution
+
+Conflict can happen in any form or from someone using hurtful and harsh language to more serious cases such as threats of voilence and everything in between.<br>Address the perceived conflict directly with those involved, preferably on a real-time basis. If this isn't possible, ask a mutual colleague to convey the message. If you are still unable to resolve the conflict, and you believe it rises to harassment or another code of conduct violation, report it.
+  
+## Reporting Violations
+
+- If you are the subject of, or witness to any violations of this Code of Conduct, please email us at <xames3.kaamiki@gmail.com> with an incident report (if possible).
```
Comment (kwattorama) [inline CODE_OF_CONDUCT.md]: Hyperlink "incident report" with incident reporting google form.

**Review by kwattorama — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (stardust-xs) [reply CODE_OF_CONDUCT.md]: Added [here](https://github.com/kaamiki/kaamiki/commit/70d1aad8ee5bcf65a38d389ef53fb58e4c9508fc), please check.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (stardust-xs) [reply CODE_OF_CONDUCT.md]: Rearranged [here](https://github.com/kaamiki/kaamiki/commit/70d1aad8ee5bcf65a38d389ef53fb58e4c9508fc).

**Review by kwattorama — APPROVED** [review]
Perfect!

**Pull request merged**